wwwroot/pages/dashboard/dashboard/js/dashboardTableComponent.js

```diff
@@ -73,6 +79,22 @@
 			ctrl.deleteModelTarget = angular.copy(powerbiReference);
 		}
 
+		ctrl.shareInit = function (powerbiReference) {
+			ctrl.shareToUsers = [];
+
+			ctrl.initUserSelect();
+
+			ctrl.shareModelTarget = angular.copy(powerbiReference);
+		}
```

Comment:
Lets rename ctrl.shareInit to much more specific function name like shareUserInit